lib/model/query/forms.js

```diff
@@ -106,9 +106,10 @@ const createVersion = (partial, form, publish = false) => ({ run, one, FormAttac
         ? def.with({ publishedAt: new Date(), xml: partial.xml })
         : def.with({ draftToken: _getDraftToken(form), xml: partial.xml })))
       .then(compose(one, insert))
-      .then(ignoringResult((savedDef) => ((publish === true)
-        ? Forms._update(form, { currentDefId: savedDef.id, name: partial.name })
-        : Forms._update(form, { draftDefId: savedDef.id, name: partial.name })))),
+      .then(ignoringResult((savedDef) =>
+        ((publish === true)
+          ? Forms._update(form, { currentDefId: savedDef.id })
+          : Forms._update(form, { draftDefId: savedDef.id })))),
```

Comment:
love this diff

Reply:
well, i'm not sure why the whitespace change but i love the rm bits

Reply:
oops, all fixed up now